resources/dicts/patrols/general/medcat.json

```diff
@@ -3844,6 +3867,28 @@
                         "amount": -5
                     }
                 ]
+            },
+			{
+                "text": "Even as {PRONOUN/p_l/poss} kit grows, p_l will always see {PRONOUN/r_c/object} as a mewling kit, always hungry for milk. p_l gazes fondly at r_c as {PRONOUN/r_c/subject} {VERB/r_c/sniff/sniffs} around, looking for herbs. It's just so wonderful seeing your kit grow up. Of course, it can be sad, but that's why p_l cherishes every moment {PRONOUN/p_l/subject} {VERB/p_l/spend/spends} with r_c.",
```

Comment:
In this patrol, p_l is the child and r_c is the parent, you'll need to flip abbrs appropriately

Reply:
fixed, thank u!!